cache.h

```diff
@@ -67,10 +76,14 @@ class Waterfield : public Heightfield {
     }
 };
 
-class NoiseField {
+class BiomeNoiseField {
 public:
-    float temperature;
+    float heat;

```

Comment:
Small nitpick: I think "temperature" is better for this, since it's more general. For example, what if we have an arctic biome? In such a case, referring to "temperature" still makes sense, but "heat" might be confusing.

Reply:
hmmm, actually I think you're right here. 👍